server/keymanapp-test-bot/emoji-label.ts

```diff
@@ -140,7 +140,7 @@ export async function processEpicLabelsEmoji(
 // Attempt to automatically add labels corresponding to issue title
 //
 // type(scope[,scope...]): title [emoji]
-// type: auto|bug|chore|docs|feat|refactor|spec|test
+// type: auto|bug|change|chore|docs|feat|refactor|spec|style|test
```

Comment:
What about cherry-pick, ci, regression?

Reply:
* cherry-pick picks up the same label as the thing it is cherry-picking
* ci is a separate label, not a type
* We don't provide question as a type, those normally belong on community site
* regression is a bug